test/e2e/gardener/shoot/create_rotate_delete.go

```diff
@@ -22,10 +22,70 @@ import (
 	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
 	e2e "github.com/gardener/gardener/test/e2e/gardener"
 	"github.com/gardener/gardener/test/e2e/gardener/shoot/internal/rotation"
+	"github.com/gardener/gardener/test/framework"
 	"github.com/gardener/gardener/test/utils/access"
 	rotationutils "github.com/gardener/gardener/test/utils/rotation"
 )
 
+func testCredentialRotation(ctx context.Context, v rotationutils.Verifiers, f *framework.ShootCreationFramework) {
```

Comment:
Please change to `...WithOffset` now that expectations are in this separate function (read https://github.com/gardener/gardener/blob/master/docs/development/testing.md for more info)